weasyl/templates/submit/character.html

```diff
@@ -57,7 +57,7 @@ <h3>Description</h3>
 
     <div class="column tagging">
         <h3>Tags</h3>
-        <textarea name="tags" class="input tags-textarea" rows="3"></textarea>
+        <textarea name="tags" class="input tags-textarea" rows="3" required></textarea>
```

Comment:
See comment on ``required`` for the modification to the journal code; I don't think we actually enforce this anywhere.